src/cogs/ownerCmds/ownerCmds.ts

```diff
@@ -1,137 +1,233 @@
 import { IModule } from "@sb-types/ModuleLoader/Interfaces";
-import { Plugin } from "../plugin";
 import { Message } from "discord.js";
 import { generateLocalizedEmbed } from "@utils/ez-i18n";
-import { EmbedType, escapeDiscordMarkdown, getMessageMemberOrAuthor } from "@utils/utils";
+import { EmbedType, escapeDiscordMarkdown, getMessageMemberOrAuthor, getUserDisplayName } from "@utils/utils";
 import { default as fetch } from "node-fetch";
-import { parse, commandRedirect } from "@utils/command";
-import * as logger from "loggy";
+import { createRedirector } from "@utils/command";
+import { ErrorMessages } from "@sb-types/Consts";
+import * as MF from "@cogs/cores/messagesFlows";
+import * as getLogger from "loggy";
 
-class OwnerCommands extends Plugin implements IModule {
+const EMOJI_OK = "👌";
+const EMOJI_FAIL = "🚫";
+
+class OwnerCommands implements IModule {
 	public get signature() {
 		return "snowball.core_features.ownercmds";
 	}
 
-	private readonly _log: Function = logger("OwnerCMDs");
+	private static _log = getLogger("OwnerCommands");
+
+	private _handler: MF.IPublicFlowCommand;
+	private _isUnloaded = false;
+
+	public async init() {
+		if (!$modLoader.isPendingInitialization(this.signature)) {
+			throw new Error(
+				ErrorMessages.NOT_PENDING_INITIALIZATION
+			);
+		}
 
-	constructor() {
-		super({
-			"message": (msg: Message) => this._onMessage(msg)
+		const redirect = createRedirector<MF.IMessageFlowContext>({
+			"change_name": (ctx) => OwnerCommands._changeName(ctx),
+			"change_avatar": (ctx) => OwnerCommands._changeAvatar(ctx)
 		});
+
+		const mfKeeper = $modLoader.findKeeper<MF.MessagesFlows>(
+			"snowball.core_features.messageflows"
+		);
+
+		if (!mfKeeper) {
+			throw new Error("MessagesFlows Keeper not found");
+		}
+
+		mfKeeper.onInit((mf) => {
+			const handler = mf.watchForCommands(
+				redirect,
+				["set_avatar", "set_username"]
```

Comment:
On above it is `change_name`, but here is `set_username`. Same for the avatar. Oops!